static/js/chart/draw_d3_map.ts

```diff
@@ -516,20 +516,23 @@ export function addMapPoints(
   getTooltipHtml?: (place: NamedPlace) => string,
   minDotRadius?: number
 ): d3.Selection<SVGCircleElement, MapPoint, SVGGElement, unknown> {
-  // get the smallest diagonal length of a region on the d3 map.
-  let minRegionDiagonal = Number.MAX_VALUE;
-  d3.select(containerElement)
-    .select(`#${MAP_GEO_REGIONS_ID}`)
-    .selectAll("path")
-    .each((_, idx, paths) => {
-      const pathClientRect = (
-        paths[idx] as SVGPathElement
-      ).getBoundingClientRect();
-      minRegionDiagonal = Math.sqrt(
-        Math.pow(pathClientRect.height, 2) + Math.pow(pathClientRect.width, 2)
-      );
-    });
-  const minDotSize = minDotRadius || Math.max(minRegionDiagonal * 0.02, 1.1);
+  let minDotSize = minDotRadius;
+  if (!minDotSize) {
```

Comment:
could you add a comment for why this is important here so we don't undo this in the future

Reply:
done